lib/index.js

```diff
@@ -89,6 +103,7 @@ PostHTML.prototype.process = function (tree, options) {
   tree = options.skipParse ? tree : parser(tree)
 
   tree.options = options
+  tree.processor = this
```

Comment:
Why is this determined in the process ? If the property list will be expanded ? if I want to instantiate a new object ? This should be in the constructor gives you more flexibility.

```js
tree.processor.constructor.name ???
```